docs/howto/odk-migrate-to-odk.md

```diff
@@ -0,0 +1,310 @@
+  instead calling:
+  ```
+  ../src/ontology/$ sh run.sh make no-mirror-refresh-imports
+  ```
+  for building all import modules at once, or:
+  ```
+  ../src/ontology/$ sh run.sh make no-mirror-refresh-%
+  ```
+  for building only a specific one.
+
+In most cases the default way of building import modules in ODK via the extraction method called SLME-BOT will
+be the best option. In some cases this default can be too “noisy” by also importing terms you don’t need/want.
+With the `module_type` & `module_type_slme` parameters in your project.yaml you can tweak the default import
+module build behavior of ODK, which relies on knowing what ROBOT can do, see also:
+* http://robot.obolibrary.org/extract
+* http://robot.obolibrary.org/filter
+* http://robot.obolibrary.org/remove
+
+The easiest ways would be to try `TOP` or `STAR` instead of the default `BOT` as value for the
+`module_type_slme` parameter and then refresh your imports to see if you still got all the terms you
+wanted, including their axioms and not any terms that are unrelated.
+
+You also have the option to use `module_type: custom` and define your own ROBOT code for building an
+import module within your custom makefile (e.g. [like this](https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/blob/main/src/ontology/vibso.Makefile#L46-L56)).
+* However, this can be quite challenging to get right, as you’d probably not want to accidentally miss
+  any asserted axioms.
+
+Check if everything worked as expected by reloading the editor file in Protégé when you refreshed an
+import module.
+
+NOTE: If you need to add a new import dependency to your _project.yaml_ (e.g. you added another
+ontology dependency to the `import_group` section), you need to run:
+   ```
+   ../src/ontology/$ sh run.sh make update_repo
+   ```
+**AND** you need to add this newly added ontology dependency also to the import declaration sections of your
+`catalog-v001.xml` and your editor file.
+
+## 8. Merge your Branch
+Once you got your import modules (dependencies) right, you can merge the branch.
+
+## 9. Make a Release
+Make a release and update the metadata needed for your purl system to resolve (e.g. OBO PURL system can also
+resolve to your import modules) see also: http://pato-ontology.github.io/pato/odk-workflows/ReleaseWorkflow/
+
+## 10. Join the ODK Slack Channel
+Come to the [#ontology-development-kit](https://obo-communitygroup.slack.com/archives/C01BKKED8R2) Slack
+channel to get help (best to have an open repo so others have something to look at for helping to fix problems
+
+## 11. Updating your ODK Environment
+To update your ODK environment follow [this HowTo](odk-update.md)).
+
+## Good to Know
+Your cheat sheet for the [Frequently Used ODK Commands](http://incatools.github.io/ontology-development-kit/FrequentlyUsedODKCommands/).
+
+## Contributors
+
+- [Philip Strömert](https://orcid.org/0000-0002-1595-3213)
+- [Nicolas Matentzoglu](https://orcid.org/0000-0002-7356-1779)
```

Comment:
Should we mention the GitHub handles as well? Given that the ODK is maintained there, the GitHub handle seems to me at least as relevant as the ORCID, maybe even more.

No strong opinion though, I’m fine with just the ORCIDs as well.

Reply:
I don't mind. I just oriented myself on some other OBOOK articles.